VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitWebviewFragment.java

```diff
@@ -59,22 +66,57 @@ public void onActivityCreated(Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
         //should we recover here?
         WebView webView = (WebView)getView().findViewById(R.id.course_unit_webView);
+        webView.getSettings().setJavaScriptEnabled(true);
         webView.setWebViewClient(new WebViewClient() {
             @Override
             public void onReceivedError(WebView view, int errorCode,
                                         String description, String failingUrl) {
-                // Handle the error
+                int error = errorCode;//for testing only
+            }
+            public void onPageFinished(WebView view, String url) {
+                view.loadUrl("javascript:EdxAssessmentView.resize(document.body.getBoundingClientRect().height)");
```

Comment:
Instead of injecting a native object, which seems like it could be potentially be a security hole, what if we fetched the value using ``webView.evaluateJavascript``